spec/requests/api/v1/reference/regions_controller_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V1::Reference::RegionsController do
+  let!(:access_token) { create(:access_token).token }
+  let(:response_json) { JSON.parse(response.body) }
+  let(:content_type) { ApiController::CONTENT_TYPE }
+  let(:headers) { { 'CONTENT_TYPE': content_type }.merge('Authorization' => "Bearer #{access_token}") }
+
+  describe 'GET /api/v1/reference/regions' do
+    let(:schema) { load_yaml_schema('get_regions_responses.yaml') }
+
+    let(:location) { create(:location) }
+    let!(:region1) { create(:region, locations: [location]) }
+    let!(:region2) { create(:region, locations: [location]) }
```

Comment:
can you add an example test that utilises the `has_and_belongs_to_many` relationship and how that looks in the response? (the 5a/5b example from the prototype)?

Reply:
Hmm.. perhaps. Let me think about how best to do that. I've kept this as simple as possible with a single location so that I don't have to get into the messiness of ordering within relations but maybe it's something that could be covered with a specific test or two.

Reply:
Added another test for this making ubiquitous use of `UnorderedArray` so shouldn't be subject to order based flakiness